test/extended/util/disruption/disruption.go

```diff
@@ -122,45 +117,68 @@ func runChaosmonkey(
 			TestData:        testData,
 			framework:       f,
 			test:            t,
-			testReport:      testCase,
+			testName:        displayName,
+			className:       "disruption_tests",
 			testSuiteReport: testSuite,
 		}
 		cm.Register(cma.Test)
 	}
 
 	start := time.Now()
 	defer func() {
-		testSuite.Update()
-		testSuite.Time = time.Since(start).Seconds()
-
-		// if the test fails and all failures are described as "Flake", create a second
-		// test case that is listed as success so the test is properly marked as flaky
-		for _, testCase := range testSuite.TestCases {
-			allFlakes := len(testCase.Failures) > 0 && len(testCase.Errors) == 0 && len(testCase.Skipped) == 0
-			for _, failure := range testCase.Failures {
-				if failure.Type == "Flake" {
-					failure.Type = "Failure"
-				} else {
-					allFlakes = false
-				}
+
+		// Calculate NumFailed and NumSkipped
+		for _, tc := range testSuite.TestCases {
+			testSuite.NumTests++
+			if tc.FailureOutput != nil {
+				testSuite.NumFailed++
 			}
-			if allFlakes {
-				testSuite.TestCases = append(testSuite.TestCases, &junit.TestCase{
-					Name:      testCase.Name,
-					Classname: testCase.Classname,
-					Time:      testCase.Time,
-				})
+			if tc.SkipMessage != nil {
+				testSuite.NumSkipped++
 			}
 		}
 
+		testSuite.Duration = time.Since(start).Seconds()
+
+		// I cannot find *anything* adding TestCases with type Flake, only TestSummaries.
```

Comment:
This is delicate, but I do not believe anything was adding k8s TestCase with type=Flake, not to be confused with TestSummaries of type flakeSummary, which was used in one place over in the alert test calling FrameworkFlakef.